pkgs/net-doc/net/scribblings/server.scrbl

```diff
@@ -0,0 +1,176 @@
+#lang scribble/doc
+
+@(require "common.rkt"
+          scribble/example
+          (for-label net/server
+                     racket/contract
+                     racket/tcp
+                     racket/list))
+
+@title[#:tag "server"]{General-purpose Server}
+
+@defmodule[net/server]{The @racketmodname[net/server] library provides
+support for running general-purpose networked servers.}
+
+@(define ev (make-base-eval))
+
+@defproc[(start-server [host string?]
+                       [port (integer-in 0 65535)]
+                       [handle (-> input-port? output-port? any)]
+                       [#:reuse? reuse? any/c #t]
+                       [#:max-allow-wait max-allow-wait exact-nonnegative-integer? 511]
```

Comment:
Yes, I remember being baffled a year or two ago when I was profiling something and the `backlog` argument wasn't working as I expected. Finally, I opened the manual for `listen` and saw:

> BUGS
>     The backlog is currently limited (silently) to 128.

I think I'll just change it so that it matches the default value of `tcp-listen`, namely 4.